docker/docker-compose.ubuntu.xenial.yml

```diff
@@ -0,0 +1,7 @@
+# Docker compose file to override configuration for centos based images
```

Comment:
Is this a standard override mechanism or how is this file used?

Reply:
Is this a standard override mechanism or how is this file used?

Yes, cf. https://docs.docker.com/compose/extends/#example-use-case

But guess the files could be generated dynamically given that they only specify which Dockerfile to use